### ISSUE TITLE
Installation problem with "node tools/build.js"

### ISSUE THREAD

**Issue by aemal** [amcat/amcat#545]
Hello!

I am getting the following error while executing "node tools/build.js :common":

```
let commander = require('commander');
^^^
SyntaxError: Unexpected strict mode reserved word
    at Module._compile (module.js:439:25)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Function.Module.runMain (module.js:497:10)
    at startup (node.js:119:16)
    at node.js:935:3

```

Thus, there is no highlightjs folder but highlight. Is it an issue?

Regards,

**Comment by vanatteveldt** [comment]
@martijnbastiaan / @CJStuart  any idea?

**Comment by CJStuart** [comment]
I believe we can just drop the entire highlightjs dependency. It's currently only used in one line of code, where it's use throws an error because it's not imported correctly. It seems a bit pointless to have a library just for highlighting code in a single debug screen, especially when it keeps breaking. 

As for the failed build of the library, I don't think it is going to cause any problems.

**Comment by CJStuart** [comment]
fixed in 9a49b3b
